pyhoma/models.py

```diff
@@ -134,11 +134,17 @@ def __len__(self) -> int:
 class State:
     __slots__ = "name", "value", "type"
 
-    def __init__(self, name: str, value: str, type: int, **_: Any):
+    def __init__(self, name: str, type: int, value: str = None, **_: Any):
         self.name = name
         self.value = value
         self.type = DataType(type)
 
+    def __getitem__(self, key):
```

Comment:
Why add this method? It's for mimic the list behavior.

Reply:
cause I thought it was needed by the States.__setitem__

Reply:
This is not required. You can remove it.